content/Arduino Cloud/Arduino Chrome App/Use-Arduino-with-Chromebook.md

```diff
@@ -65,21 +60,13 @@ Follow these steps:
 
 1. Connect a [compatible board](https://support.arduino.cc/hc/en-us/articles/360014779899#chromebook) to one of the USB ports on your Chromebook.
 
-1. In the board selector menu, choose **Select Other Board & Port**
-
-   ![The Boards Dropdown in the Cloud Editor.](img/web-editor-boards-dropdown.png)
-
-1. Pick the specific board you're using and click **OK**.
-
-1. Click the **Upload** button ![Upload button](img/upload_button.png) in the editor.
-
-1. That should open a **Authorise USB connection** window. Click **Select port**.
+1. In the board selector menu, click on the **Detect Device** button.
 
-   ![Authorise USB connection](img/Authorise-USB-connection.png)
```

Comment:
`img/Authorise-USB-connection.png` is now unused so the file should be removed from the repository.